flink-shaded-asm-9/src/main/resources/META-INF/NOTICE

```diff
@@ -1,4 +1,4 @@
-flink-shaded-asm7
+flink-shaded-asm9
```

Comment:
@zentol Out of curiosity: what is this value referring to? Initially, I thought that it's the modules name (which would have meant that the hyphen between `asm` and `9` is missing). But looks like this line does not match the module name in several places (e.g. `flink-shaded-zookeeper-3` for `flink-shaded-zookeeper-35` and `flink-shaded-zookeeper-34`).